diff --git a/node_modules/@supabase/supabase-js/dist/index.cjs b/node_modules/@supabase/supabase-js/dist/index.cjs
index 56a9c3a..df157ae 100644
--- a/node_modules/@supabase/supabase-js/dist/index.cjs
+++ b/node_modules/@supabase/supabase-js/dist/index.cjs
@@ -624,7 +624,7 @@ var require_extract = /* @__PURE__ */ __commonJSMin(((exports) => {
 	let otelModulePromise = null;
 	const OTEL_PKG = "@opentelemetry/api";
 	function loadOtel() {
-		if (otelModulePromise === null) otelModulePromise = Promise.resolve(`${OTEL_PKG}`).then((s) => tslib_1.__importStar(require(s))).catch(() => null);
+		if (otelModulePromise === null) otelModulePromise = Promise.resolve(null);
 		return otelModulePromise;
 	}
 	/**
diff --git a/node_modules/@supabase/supabase-js/dist/index.mjs b/node_modules/@supabase/supabase-js/dist/index.mjs
index 0b33345..fc81723 100644
--- a/node_modules/@supabase/supabase-js/dist/index.mjs
+++ b/node_modules/@supabase/supabase-js/dist/index.mjs
@@ -68,7 +68,7 @@ function __awaiter(thisArg, _arguments, P, generator) {
 let otelModulePromise = null;
 const OTEL_PKG = "@opentelemetry/api";
 function loadOtel() {
-	if (otelModulePromise === null) otelModulePromise = import(/* webpackIgnore: true */ /* turbopackIgnore: true */ /* @vite-ignore */ OTEL_PKG).catch(() => null);
+	if (otelModulePromise === null) otelModulePromise = Promise.resolve(null);
 	return otelModulePromise;
 }
 /**
